<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="start_Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>450</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 204, 102);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>801</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#66ccff;&quot;&gt;Welcome to the Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="load_Button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>450</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 128, 128);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QLabel" name="scoreboard">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>521</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="manufactured">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+1000 pencils
 manufactuerd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>510</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BuyGraphite">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(160, 26, 26);
</string>
      </property>
      <property name="text">
       <string>Buy Graphite
</string>
      </property>
     </widget>
     <widget class="QLabel" name="Inventory">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Pencils In Inventory: 0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>510</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Debugging tool</string>
      </property>
     </widget>
     <widget class="QLabel" name="PencilsProduced">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Pencils produced (hh : mm : ss) : 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="PublicDemand">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Public Demand: 1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="PencilRate">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>350</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Rate: 120 pencils / minute</string>
      </property>
     </widget>
     <widget class="QLabel" name="PencilPrize">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Price of Pencil: $1.00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="woodGranite">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+10 m of
 Wood and Graphite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BuyMore">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>430</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Buy More
</string>
      </property>
     </widget>
     <widget class="QLabel" name="WoodLeft">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Wood Left: 1000.00 m</string>
      </property>
     </widget>
     <widget class="QLabel" name="Intelligence">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Bank Balance&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="GraphiteLeft">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>376</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Graphite Left: 1000.00 m</string>
      </property>
     </widget>
     <widget class="QLabel" name="AutoPencil">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>380</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>AutoPencil2000: 0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="marketing">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>300</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Unlock Marketing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>510</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QLabel" name="ProgramName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Pencil Producer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MakePencil">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>70</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Make Pencil</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DecreasePrize">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>224</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IncreasePrize">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>224</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="inventory">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>224</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+1000 pencils
 in inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upgradeApm">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Upgrade</string>
      </property>
     </widget>
     <widget class="QLabel" name="buy_graphite_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>$1500.00/100.00m</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="grantMoney">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>224</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+ $1000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BuyWood">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(160, 26, 26);
</string>
      </property>
      <property name="text">
       <string>Buy Wood
</string>
      </property>
     </widget>
     <widget class="QLabel" name="BankBalance">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(78, 154, 6);
font: 24pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>$145.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="buy_wood_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>$1000.00/100.00m</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="buy_more_label">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>460</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>$ 150.00</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
